elastic/kibana
# [ILM] Add missing deprecated param to API schema

## Summary

When we merged https://github.com/elastic/kibana/pull/174150 we forgot to allow the `deprecated` param in the API. This PR addresses this issue. 

##### How to test

* Start up kibana and es
* Navigate to Stack management -> ILM and enable `deprecated` policies to be shown
* Edit any of them and make a small change in it
* Verify that saving that policy is successful and the changes are reflected in the UI
